.changeset/early-keys-know.md

```diff
@@ -0,0 +1,10 @@
+---
+"@lens-protocol/blockchain-bindings": minor
+"@lens-protocol/api-bindings": minor
+"@lens-protocol/domain": minor
+"@lens-protocol/react": minor
+"@lens-protocol/react-native": minor
+"@lens-protocol/react-web": minor
+---
+
+**feat:** added `useSignFrameAction` hook
```

Comment:
Don't we need a client changeset too?

Reply:
I skipped it here as this PR merges into the client SDK one, and there is a changeset for client already there 

https://github.com/lens-protocol/lens-sdk/pull/887/files#diff-270dee66ee89cf46e8101f30e11bc690d809f1bf8943c25c0b4e36e0a7196e5b